polkadot-wiki/sidebars.js

```diff
@@ -130,7 +129,7 @@ module.exports = {
                 "learn/learn-common-goods",
                 "learn/learn-auction",
                 "learn/learn-crowdloans",
-                "learn/learn-teleport",
```

Comment:
@filippoweb3 did you intend to remove `learn/learn-teleport` also?  I just see the `learn/learn-security` doc removed  in this PR and replaced with `learn/learn-parachains-faq`

Reply:
yes as the same page is present in Basics/Tokens and Assets